parser.py

```diff
@@ -9,66 +9,47 @@ def headlines(design = "APEX"):
 
     # split into physical attributes and simulation results
     rocket = root[0]
-    sims = root[1]
-
+    sims   = root[1]
+
+    def simdata(rootObj,dataset,att):
+        try:
+            res = rootObj[0].find(dataset).attrib[att]
+        except: 
+            res = "Error parsing simulation/rocket data"
+        return res 
+    
     # get motor
-    try:
-        for element in rocket.iter('motorconfiguration'):
-            motor = element[0].text
-    except:
-        motor = "No motor found"
+    motor   = "motor not found"
+    element = rocket.iter('motorconfiguration')
```

Comment:
Maybe this should be a for loop? @shreybohra

Reply:
yeah iter method needs to be used in a for loop as far as I know